android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -51,8 +56,12 @@
 	private float shadowX = 0f;
 	private float shadowY = 0f;
 	private int shadowColor = Color.TRANSPARENT;
-	private String minimumFontSize = null;
-	private String autoshrinkSetFontSize = null;
+	private int autoLinkFlags;
+	private int viewHeightInLines;
+	private int maxLines = Integer.MAX_VALUE;
+	private float minimumFontSizeInPixels = -1.0f;
```

Comment:
Yeah, I mean we should allow `minimumFontSize` to accept `Number/String` just like `fontSize`.

Also note, the docs say "_On Android the default font sizing is in pixels_" for `fontSize`, this needs fixing.